src/routes/docs/products/sites/site-hosting/ssr/+page.markdoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: article
+title: Server Site Rendering
+description: Learn how to host SSR web apps on Appwrite Sites.
+---
+
+Server Side Rendering (SSR) apps generate HTML content dynamically on the server for each request and send fully rendered pages to the browser. This approach improves performance for the initial load and enhances SEO since search engines can easily index the content. While SSR can be slightly slower than static apps due to server-side processing, it provides a good balance between performance and interactivity.
```

Comment:
I miss some connection to our edge network. I would add here some explanation of the Appwrite CDN, Edge, Region topology with relevant links. Bottom line: we have a CDN, your executions will happen on the nearest regions to the user, your data is stored in your region of choice.

Reply:
Will work on this

Reply:
Added, would definitely like a review